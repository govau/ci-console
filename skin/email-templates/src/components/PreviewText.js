import React from "react";
import styled from "styled-components";

const Span = styled.span`
  display: none;
  font-size: 0px;
  line-height: 0px;
  max-height: 0px;
  max-width: 0px;
  opacity: 0;
  overflow: hidden;
  visibility: hidden;
  msohide: all;
`;

export default ({ children }) => <Span>{children}</Span>;
