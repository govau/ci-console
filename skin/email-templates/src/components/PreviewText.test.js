import React from "react";
import renderer from "react-test-renderer";

import PreviewText from "./PreviewText";

test("renders without crashing", () => {
  const tree = renderer.create(<PreviewText>preview</PreviewText>);
  expect(tree).toMatchSnapshot();
});
