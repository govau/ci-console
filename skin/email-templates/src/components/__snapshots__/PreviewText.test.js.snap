// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
.c0 {
  display: none;
  font-size: 0px;
  line-height: 0px;
  max-height: 0px;
  max-width: 0px;
  opacity: 0;
  overflow: hidden;
  visibility: hidden;
  msohide: all;
}

<span
  className="c0"
>
  preview
</span>
`;
